randovania/games/samus_returns/pickup_database/pickup-database.json

```diff
@@ -204,7 +211,9 @@
             "broad_category": "beam_related",
             "model_name": "powerup_grapplebeam",
             "offworld_models": {
-                "dread": "powerup_grapplebeam"
+                "dread": "powerup_grapplebeam",
+                "prime1": "Grapple Beam",
+                "prime2": "GrappleBeam"
```

Comment:
missing am2r?

Reply:
The grapple beam that dcr wished for is only available in yams 2.1.0 ( #6524 )

Reply:
I probably can add it anyway if yams is gonna be updated in rdv soon.

Reply:
Then you can also add the other items:
```json
    "sItemBabyMetroidMSR": 0.2,
    "sItemBeamBurstMSR": 0.2,
    "sItemLightningArmorMSR": 0.2,
    "sItemPhaseDriftMSR": 0.2,
    "sItemScanPulseMSR": 0.2,
    "sItemSpiderBallMSR": 0.2,
```

Reply:
Added the grapple beam